.github/workflows/ci.yml

```diff
@@ -116,7 +116,7 @@ jobs:
           pip install --progress-bar=off --upgrade --requirement tests/requirements-tools.txt
 
       - name: Install test dependencies (libraries)
-        if: ${{ !endsWith(matrix.python-version, '-dev') }}
+        if: matrix.python-version != '3.12'
```

Comment:
The "proper" way would be to use `python-version: [..., '3.12-dev'] until the release and until we can install full environment.